app/controllers/escorts_controller.rb

```diff
@@ -30,6 +30,11 @@ def cancel
     end
   end
 
+  def approve
```

Comment:
Consider a pre trigger at the top to authenticate sergeants maybe

Reply:
👍